addons/website_slides/static/src/scss/slides_slide_fullscreen.scss

```diff
@@ -92,17 +92,32 @@
 
             .o_wslides_sidebar_content {
                 line-height: unset;
+                font-weight: 500;
```

Comment:
Same remark for CSS: some font-weight could probably be replaced by standard bootstrap classes, not sure for padding, but could you have a look to try to remove as much as possible ? The less css we have, the more easy it is to adapt to custom themes, responsive, ... :)